client/src/helpers/test/__snapshots__/ClassList.spec.js.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ClassName test Basic testing renders as expected 1`] = `
+<div>
+  <div class="classlist-header-group">
```

Comment:
Snapshot tests are good to have yet we need to have the means to easily update the snapshots.  Soon, we need to add a script to the package.json with a command that can regenerate the snapshots after intentionaly changes.

Reply:
no need to make any changes for now regarding snapshots

Reply:
Noted. Thank you for the feedback!